### PR TITLE
KV Store: Graveler naive delete repository

### PR DESCRIPTION
**Pull request by N-o-Z**

### CODE REVIEW

File: pkg/graveler/graveler.go [hunk 1]
```diff
@@ -892,7 +895,71 @@ func (g *KVGraveler) WriteMetaRangeByIterator(ctx context.Context, repositoryID
...
+	defer itr.Close()
+	var wg multierror.Group
+	for itr.Next() {
+		commit := itr.Value()
+		wg.Go(func() error {
+			return g.RefManager.RemoveCommit(ctx, repositoryID, commit.CommitID)
+		})
+	}
+	return wg.Wait().ErrorOrNil()
+}
```
Comment (itaiad200) [inline pkg/graveler/graveler.go:944]: I'm unsure that creating (#Tags + #Commits + #Branches) goroutines is a good idea. You may be needing workers to make sure you're not creating a 1M routines for large repositories.

**Review by itaiad200 — APPROVED** [review]
Added a single comment, let's make sure we control the load for the deletion

File: pkg/graveler/graveler.go (hunk 1 above)
Comment (N-o-Z) [reply pkg/graveler/graveler.go:944]: @itaiad200  Do you think it's better to just serialize the calls for now? This is just temporary solution, which obviously we will not release KV with

Comment (itaiad200) [reply pkg/graveler/graveler.go:944]: For a temporary solution it's fine for now.

**Pull request merged**